### DISCUSSION TITLE
New UI Published 🚀🤩

### DISCUSSION THREAD

**Discussion by Spyware007** [Spyware007/Animating-Buttons#1556] [Announcements]
<img width="1440" alt="Screenshot 2023-06-25 at 4 41 34 AM" src="https://github.com/Spyware007/Animating-Buttons/assets/89961974/0f55c12f-8fa9-440b-b9ee-26d57b1a4204">

## New Features: 🚀
1. Login using GitHub .
2. You can add the code of buttons directly from the website.
3. User Profile.
4. Like counter for each button.
5. Firebase added for the backend.

## What's next? 😎
1. Leaderboard.
2. About Project page.
3. Explore section for some of the best buttons.

<hr /><em>This discussion was created from the release <a href='https://github.com/Spyware007/Animating-Buttons/releases/tag/v1.0.0'>New UI Published 🚀🤩</a>.</em>

**Comment by Riyazul555** [comment]
This UI looks nice @Spyware007
